e2e/pages/home.page.ts

```diff
@@ -0,0 +1,31 @@
+import { Locator, Page } from "@playwright/test"
+
+export class HomePage {
+  readonly page: Page
+  readonly mentorsButton: Locator
+  readonly loginButton: Locator
+  readonly availableMentorsList: Locator
+  readonly featuredMenteesList: Locator
+
+  constructor(page: Page) {
+    this.page = page
+    this.mentorsButton = page.getByRole("link", { name: /discover mentors/i })
+    this.loginButton = page.getByRole("link", { name: /login to continue/i })
+    this.availableMentorsList = page.locator(
```

Comment:
Please use the `getBy` query as much as possible, following [this priority](https://testing-library.com/docs/queries/about/#priority).

Reply:
updated https://github.com/bearmentor/bearmentor/pull/8/commits/5d078d7fb66a9ce09a715fbb88b0c046ed7a9f44